modules/core/shared/src/main/scala/lucuma/core/enum/GmosNorthFpu.scala

```diff
@@ -35,13 +35,13 @@ object GmosNorthFpu {
   /** @group Constructors */ case object LongSlit_1_50 extends GmosNorthFpu("LongSlit_1_50", "1.5\"", "Longslit 1.50 arcsec", Some(Angle.fromDoubleArcseconds(1.50)), Angle.fromDoubleArcseconds(0.000))
   /** @group Constructors */ case object LongSlit_2_00 extends GmosNorthFpu("LongSlit_2_00", "2.0\"", "Longslit 2.00 arcsec", Some(Angle.fromDoubleArcseconds(2.00)), Angle.fromDoubleArcseconds(0.000))
   /** @group Constructors */ case object LongSlit_5_00 extends GmosNorthFpu("LongSlit_5_00", "5.0\"", "Longslit 5.00 arcsec", Some(Angle.fromDoubleArcseconds(5.00)), Angle.fromDoubleArcseconds(0.000))
-  /** @group Constructors */ case object Ifu1 extends GmosNorthFpu("Ifu1", "IFU-2", "IFU 2 Slits", Option.empty[Angle], Angle.fromDoubleArcseconds(33.500))
-  /** @group Constructors */ case object Ifu2 extends GmosNorthFpu("Ifu2", "IFU-B", "IFU Left Slit (blue)", Option.empty[Angle], Angle.fromDoubleArcseconds(31.750))
-  /** @group Constructors */ case object Ifu3 extends GmosNorthFpu("Ifu3", "IFU-R", "IFU Right Slit (red)", Option.empty[Angle], Angle.fromDoubleArcseconds(35.250))
+  /** @group Constructors */ case object Ifu2 extends GmosNorthFpu("Ifu2", "IFU-2", "IFU 2 Slits", Option.empty[Angle], Angle.fromDoubleArcseconds(33.500))
```

Comment:
We might go with `Ifu2Slits` as Rob suggested because `Ifu2` was the old name for `IfuBlue`?

Reply:
Sounds good, `Ifu2Sllits` seems good